hack/gen-swagger-doc/gen-swagger-docs.sh

```diff
@@ -63,6 +63,9 @@ if [ "$OUTPUT_FORMAT" = "html" ]; then
     # $$ has special meaning in asciidoc, we need to escape it
     sed -i 's|\$\$|+++$$+++|g' "$WORKDIR/definitions.adoc"
     sed -i '1 i\:last-update-label!:' "$WORKDIR/"*.adoc
+    sed -i -e "/KubeVirt API\$/a\\:revnumber: $(git rev-parse HEAD || echo no-revision)" \
+            -e "/__Terms of service__ :/a\\__Version__ : {revnumber}" \
```

Comment:
Maybe we can make it even a link to the core repo?

Reply:
Done